usrtools/txgen/app/latency.c

```diff
@@ -0,0 +1,296 @@
+/*-
+ * Copyright(c) <2016-2021>, Intel Corporation. All rights reserved.
```

Comment:
Please adjust the copyrights to have the SPDX as the first line and remove the ',' after  the dates we should have fixed this everywhere in CNDP. You can leave the comma and we need to find and remove the comma in all of CNDP. This makes it easier to search and update the copyright.

I.E.,

/* SPDX-License-Identifier: BSD-3-Clause
 \* Copyright (c) 2019-2022 Intel Corporation.
 \* Copyright (c) 2022 Red Hat, Inc.
 \*/

The format is a bit messed up, but I could not get it to align the text without indenting it.